### DISCUSSION TITLE
how to set global font bold style?

### DISCUSSION THREAD

**Discussion by Yurii-huang** [Textualize/textual#4780] [Q&A] [answered]
how to set global font bold style?

**Comment by TomJGooding** [comment, marked as answer]
https://textual.textualize.io/guide/CSS/#universal-selector

**Reply by Yurii-huang** [reply]
```    
* {
        text-style: bold;
        border-title-style: bold;
    }
```

**Reply by Yurii-huang** [reply]
How should I set the background color for Text()? @TomJGooding 

![image](https://github.com/user-attachments/assets/d14e1cf5-0e3b-4b09-9d02-90f603d67a4a)

**Comment by Yurii-huang** [comment]
style=Style(bold=True, bgcolor="green")
